.mailmap

```diff
@@ -16,87 +16,42 @@
 #
 Abdón Rodríguez Davila <a@abdonrd.com> <a@abdonrd.com>
 Abigail J. Cross <across@us.ibm.com>
-Abraham Asfaw <asfaw@princeton.edu>
-Adenilton Silva <adjs@users.noreply.github.com>
-Adrian Chen <adrian.evangelion@gmail.com>
 Albert Frisch <albert.frisch@de.ibm.com>
 Albert Frisch <albert.frisch@de.ibm.com> <alfr@de.ibm.com>
-Aleksei Malyshev <aleksei.o.malyshev@gmail.com>
 Ali Javadi-Abhari <ali.javadi@ibm.com> <ajavadia@users.noreply.github.com>
 Ali Javadi-Abhari <ali.javadi@ibm.com> <ajavadia@princeton.edu>
 Andrew W. Cross <awcross@us.ibm.com>
 Andrew W. Cross <awcross@us.ibm.com> Andrew Cross <awcross@Andrews-MacBook-Pro.local>
 Andrew W. Cross <awcross@us.ibm.com> Andrew Cross <awcross@andrews-mbp.watson.ibm.com>
 Anna Phan <9410731+attp@users.noreply.github.com> <anna.phan@au1.ibm.com>
-Anton Karazeev <anton.karazeev@gmail.com>
 Antonio D. Córcoles-Gonzales <adcorcol@us.ibm.com>
 Antonio Mezzacapo <30698465+antoniomezzacapo@users.noreply.github.com> <amezzac@us.ibm.com>
-AzizNgoueya <49644465+AzizNgoueya@users.noreply.github.com>
-Borja Godoy Gago <Borja.Godoy.Gago@gmail.com>
-Bruno Schmitt <bruno.schmitt@epfl.ch>
-Cameron McGarry <cameron@cmcgarry.co.uk>
 Carmen Recio Valcarce <car_rv@outlook.es> <49230820+carmenrv@users.noreply.github.com>
-Charles Taylor <charlesastaylor@gmail.com>
 Chris Culver <chris.cb.culver@gmail.com>
 Christian Claus <cclauss@bluewin.ch>
-Christian Clauss <cclauss@me.com>
```

Comment:
I think these 2 lines might be the same person and probably needs to be kept but updated to correctly map the commit authors as the same person. Probably something like:

```
Christian Clauss <cclauss@me.com> <cclauss@bluewin.ch>
```

@cclauss can you confirm?

Reply:
Bluewin is dead...  Long live me.com.

Reply:
done!